invalid_sample_generator.py

```diff
@@ -0,0 +1,153 @@
+import requests
+import csv
+import xml.etree.ElementTree as ET
+import json
+import random
+import os
+
+
+biosample_field_vals = {
+    "collection date":"collection_date"
```

Comment:
This solves only the collection date issue. Would you have a problem with other synonym fields?

Reply:
we have to manually go through all files to understand the mismatched keys unless biosample already have it. I manually checked few checklists I could not find any other keys mismatching, let's run it and fix it if fails.

Reply:
Well, it does not sound like the ideal solution. Why not look for any of the synonyms in the json document?